lib/vagrant-google/action/run_instance.rb

```diff
@@ -62,6 +63,11 @@ def call(env) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
           autodelete_disk     = zone_config.autodelete_disk
           service_accounts    = zone_config.service_accounts
 
+          # If image_family is set, get the latest image image from the family.
+          if !image_family.nil?
```

Comment:
What if both `image` and `image_family` are set? I suggest failing with error. 

See how the preemptible does it here: https://github.com/mitchellh/vagrant-google/blob/9a45af32a6ef298c86dcf9b71d98b91279416d64/lib/vagrant-google/config.rb#L370